### ISSUE TITLE
New kind of bloatwarte?

### ISSUE THREAD

**Issue by agarzon** [Disassembler0/Win10-Initial-Setup-Script#169]
Look like there is a new kind of pre-installed apps on W10 and those are harder to remove by scripting.

I just installed W10, updated all and run this powershell script. it clean up and improve the RAM use from 63% ro 53% (yes, it's a old laptop) :blush:

But I noticed some icons remaining in the bar that were not removed,  **Township**, **Drawboard PDF** and **Adobe Photoshop elements 2018**

After running ```Get-AppxPackage -AllUsers | Select Name, PackageFullName``` they are not listed but I notice some apps using hashed names instead regular names, here:

- 1527c705-839a-4832-9118-54d4Bd6a0c89_10.0.17134.1_neutral_neutral_cw5n1h2txyewy
- c5e2524a-ea46-4f67-841f-6a9465d9d515_10.0.17134.1_neutral_neutral_cw5n1h2txyewy
- E2A4F912-2574-4A75-9BB0-0D023378592B_10.0.17134.1_neutral_neutral_cw5n1h2txyewy
- F46D4000-FD22-4DB4-AC8E-4E1DDDE828FE_10.0.17134.1_neutral_neutral_cw5n1h2txyewy

After google a bit i found I'm not the only one and that guy had the same App Id's:

https://www.reddit.com/r/PowerShell/comments/9uxqis/windows_10_app_uninstall_1803/

Sorry I cannot help more, I'm more a Linux guy. :unamused:

**Comment by Disassembler0** [comment]
This topic has been discussed numerous times. For more details, see #8, #38, #101, #145.

**tl;dr:** There are apps which are *installed* (*provisioned*) and apps which are *suggested*. The provisioned ones can be uninstalled and that's what the script does. The suggested ones can't be uninstalled (because they are not installed) and have just a placeholder tile which runs the installation when user clicks on it. The extraneous icons for suggestions can be removed using `UnpinStartMenuTiles` tweak which is not included in the default preset because it irreversibly unpins all icons and if you want to revert, you have to do so manually.

As for the weird names, those are in fact GUIDs, and if the OP in the reddit thread would read the output of `Get-AppXPackages` a bit further, they would see that:

| GUID | Name |
|-|-|
| 1527c705-839a-4832-9118-54d4Bd6a0c89 | Microsoft.Windows.FilePicker |
| c5e2524a-ea46-4f67-841f-6a9465d9d515 | Microsoft.Windows.FileExplorer |
| E2A4F912-2574-4A75-9BB0-0D023378592B | Microsoft.Windows.AppResolverUX |
| F46D4000-FD22-4DB4-AC8E-4E1DDDE828FE | Microsoft.Windows.AppSuggestedFoldersToLibraryDialog |

So nothing to worry about.
